react/utils/validation.ts

```diff
@@ -1,11 +1,15 @@
 export const CPF_MASK = '999.999.999-99'
 
-export const unmaskCPF = (value: string) => value.replace(/\D/g, '')
+const unmaskCPF = (value: string) => value.replace(/\D/g, '')
 
 export const validateCPF = (value: string) => {
   const unmaskedValue = unmaskCPF(value)
 
-  if (unmaskedValue.length < 11) {
+  if (unmaskedValue.length !== 11) {
```

Comment:
Why 11?

Reply:
it's the maximum number of digits in a CPF

Reply:
Maybe using a `CPFLength` variable for this? Even if most of our devs are BR based, this might fall into a non BR maintainer in the future. What do you think?

Reply:
done in d508fe0 :-)